exasol_script_languages_container_tool/lib/api/upload.py

```diff
@@ -55,13 +56,9 @@ def upload(
     ssl_cert_path: str = "",
     use_ssl_cert_validation: bool = True,
 ) -> luigi.LocalTarget:
-    """
-    This command uploads the whole script-language-container package of the flavor to the database.
-    If the stages or the packaged container do not exists locally, the system will build, pull or
-    export them before the upload.
-    :raises api_errors.TaskFailureError: if operation is not successful.
-    :return: Path to resulting report file.
-    """
```

Comment:
documentation should stay anyway I think

Reply:
The Python standard libraries remove the documentation when a method becomes _deprecated_:
https://github.com/python/cpython/blob/8aaf7525ab839c32966ee862363ad2543a721306/Lib/logging/__init__.py#L1534

Reply:
ok, then is fine